x-pack/test/functional/apps/ml/anomaly_detection_integrations/single_metric_viewer_dashboard_embeddables.ts

```diff
@@ -83,14 +83,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           );
         });
 
-        it('create new single metric viewer panel', async () => {
+        it('should create new single metric viewer panel', async () => {
           await ml.dashboardEmbeddables.clickSingleMetricViewerInitializerConfirmButtonEnabled();
           await PageObjects.timePicker.pauseAutoRefresh();
           await ml.dashboardEmbeddables.assertDashboardPanelExists(testData.panelTitle);
           await ml.singleMetricViewer.assertChartExist();
-          await ml.singleMetricViewer.assertAnomalyMarkerExist();
           await PageObjects.dashboard.saveDashboard(testData.dashboardTitle);
         });
+
+        it('should have anomaly click action menu', async () => {
+          await ml.dashboardEmbeddables.assertDashboardPanelExists(testData.panelTitle);
+          await ml.singleMetricViewer.assertAnomalyMarkerExist();
+          await ml.singleMetricViewer.openAnomalyMarkerActionsPopover();
+          await ml.singleMetricViewer.assertAnomalyActionDiscoverButtonExists();
+          await ml.singleMetricViewer.assertAnomalyActionJobRulesButtonExists();
```

Comment:
Would be nice to verify that the click opens the rules flyout, but I don't think we are currently doing this for the anomalies table tests either.

Reply:
Added in 80b982d1a1d0def149d03f25c31631daf2b09df1